image-builder/image_builder.sh

```diff
@@ -478,8 +486,11 @@ main() {
 	fi
 
 	if [ -n "$container_engine" ]; then
-		build_with_container "${rootfs}" "${image}" "${fs_type}" "${block_size}" \
-						  "${root_free_space}" "${agent_bin}" "${agent_init}" "${container_engine}"
+		build_with_container "${rootfs}" \
+			"${image}" "${fs_type}" "${block_size}" \
+			"${root_free_space}" "${agent_bin}" \
+			"${agent_init}" "${container_engine}" \
+			"${nsdax_bin}"
```

Comment:
ok, I see something wrong here, ${nsdax_bin} is the 9th element, but in `build_with_container` is the 8th (local nsdax_bin="$8"),  could you please add a new variable in `build_with_container` to save the 8th element (local container_engine ="$8") ?

Reply:
Nice catch! Sorry for not testing that path. I've updated the commit now

Reply:
thanks @crobinso, I blame @devimc for that bug